src/gui/creds/httpcredentialsgui.cpp

```diff
@@ -93,7 +92,8 @@ void HttpCredentialsGui::asyncAuthResult(OAuth::Result r, const QString &user,
         break;
     }
 
-    OC_ASSERT(_user == user); // ensured by _asyncAuth
+    // FIXME: how can we ensure this? do we even have to store the username in case of OAuth?
```

Comment:
Its already handled in the oauth class I think

Reply:
It doesn't assert it, but emits an error state. We could "upgrade" this to `OC_ENSURE`.